packages/Input/stories/examples/Showcase.js

```diff
@@ -25,8 +26,10 @@ const getKnobs = () => ({
 
 export default function Showcase() {
   return (
-    <CodeViewer>
-      <Input {...getKnobs()} />
-    </CodeViewer>
+    <InputStory storyName="Input" tagline={InputStory.defaultTaglines.showcase}>
```

Comment:
I actually prefer to have the `<ExampleStory>` wrapper in the `.stories.js` file – then if we want to use the actual story in a different context (like on the Starling site, or in the Docs MDX page) it won't include the heading, padding, tagline, etc – which would keep it simpler and give us a bit more flexibility for how we use it.